lib/bus.js

```diff
@@ -26,10 +26,12 @@ function Bus(resource) {
   this.type = 0xFFFE;
   this.subnet = parseInt(subnet);
   this.address = new Buffer([0, 0, 0, 0]);
+  //Enabling broadcast enables interacting with other devices on the IP network talking smart-bus
+  this.broadcast = config.broadcast? true : false;
```

Comment:
This was my first thought, however I thought for the effort required to make it an option, that would be the best way forward.  Your call.

Reply:
But anyway, what is the reason to broadcast udp messages if you will be able to simple create several `Bus` instances?

Reply:
Ok, `Bus` and `Gateway` classes certainly should be splitted and broadcast mode should be enabled by default.

Reply:
Shall we just leave it as an option for now?

Reply:
Let's implicitly get this property from the gateway address. If defined gateway ip is a broadcast address then initialize broadcast socket otherwise no.

Reply:
Sounds like a good idea. I suggest using a cidr library (eg https://www.npmjs.com/package/node-cidr#broadcast) as it will accurately determine this even when the subnet is classless. Shall I include this functionality in this PR?

Reply:
Solved in 
https://github.com/caligo-mentis/smart-bus/commit/02bdcc3f4db5b6a1b1c7e05e10359264092baa05